src/views/CollectionImageSlider/Page.tsx

```diff
@@ -0,0 +1,53 @@
+import arrowLeftIcon from "images/arrow-left.svg";
```

Comment:
Which page is this? How do we get to this view?

Reply:
This page is under implementation